### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

File: src/Pages/Home/index.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            <ColHeadline>
+              <Headline>
+                <TextColor>EverR</TextColor>etrofit, <br />
+                votre voiture, <br />
+                pour <TextColor>toujours</TextColor> <br />
+              </Headline>
+              <SubHeadline>
+                Vous êtes attaché à votre véhicule mais il est désormais trop
+                polluant ? Trop chère à entretenir ? Le prix du carburant
+                classique est trop élevé ? Nous avons la solution !
```
Comment (Caroline-CD) [inline src/Pages/Home/index.jsx:50]: The client's spelling mistakes will have to be corrected

**Review by Caroline-CD — APPROVED** [review]
Ok for me

File: src/App.jsx [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+import { Routes, Route } from 'react-router-dom';
+import Header from './Components/Header/index';
+import Home from './Pages/Home';
 
-export default App;
+export default function App() {
+  return (
+    <div>
+      <Header />
+      <Home />
```
Comment (Hacene010) [inline src/App.jsx:9]: I think you forgot to remove this component

File: src/Components/Header/style.jsx [hunk 3]
```diff
@@ -0,0 +1,205 @@
+import styled from 'styled-components';
+import { Home } from '@styled-icons/entypo/Home';
+import { User } from '@styled-icons/boxicons-solid/User';
+import { Bars } from '@styled-icons/fa-solid/Bars';
+
+export const SHeader = styled.div`
```
Comment (Hacene010) [inline src/Components/Header/style.jsx:6]: You must remove all these exports and put your const into the last export without the default word

File: src/Components/SearchBar/style.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { Search } from '@styled-icons/bootstrap/Search';
+
+export const Ssearchbar = styled.div`
```
Comment (Hacene010) [inline src/Components/SearchBar/style.jsx:4]: Same here for the exports

File: src/Pages/Home/index.jsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                <FullBattery />
+                <p>Une autonomie jusqu&apos;à 350km</p>
+              </Col>
+            </Row2>
+          </Container>
+        </MainContainer>
+        <TextArea>
+          Autorisé en France depuis avril 2020, le rétrofit commence de plus en
+          plus à trouver sa place sur le marché français.
+          <br /> <br />
```
Comment (Hacene010) [inline src/Pages/Home/index.jsx:95]: all these br  are ugly :/

File: src/Pages/Home/index.jsx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+          </VoteContainerRow1>
+          <VoteContainerRow2>
+            <VoteCard />
+            <VoteCard />
+            <VoteCard />
+          </VoteContainerRow2>
+        </VoteContainer>
+        <NewsContainer>
+          <NewsHeader>Actualités sur les véhicules électriques</NewsHeader>
+          <NewsCardContainer>
```
Comment (Hacene010) [inline src/Pages/Home/index.jsx:125]: a .map here is more appropriate, isn't it?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some evils. See the comments

**Pull request closed without merging**